site/_apps/zuix-hn.txt

```diff
@@ -0,0 +1,26 @@
+---
+weight: 2
+title: 'Zuix HN'
+github-title: 'genielabs/zuix-hn'
+library: 'ZUIX'
+libraries: 
+    - name: 'ZUIX'
+module-bundling: 'zuix-bundler'
+service-worker: 'Application Shell'
+perfomance-patterns: 'Lazy loaded components with caching'
+server-side-rendering: 'None'
+api: 'Hacker News Firebase API'
+hosting: 'Firebase'
+authors: 
+    - name: 'Gene'
+lighthouse: '91/100'
+interactive-em: '3.8s'
+interactive-faster-3g: '2.6s'
```

Comment:
No, this is the First interactive time as you can see from the linked WebPageTest results in this same file. Now those could be a little slower due to custom fonts inclusion.